huaweicloud/services/acceptance/nat/resource_huaweicloud_nat_dnat_rule_test.go

```diff
@@ -225,3 +225,209 @@ resource "huaweicloud_nat_dnat_rule" "test" {
 }
 `, testAccPublicDnatRule_base(name))
 }
+
+func TestAccPublicDnatRule_associatedGEIP(t *testing.T) {
```

Comment:
It seems that there is no need to add new test cases.
 Can you put the tests for the new fields into the original test cases?